### PR TITLE
Lab1

### PR DESCRIPTION
**Pull request by keirashea**
@molexted 
@oliviatessa

### CODE REVIEW

**Comment by no-lex** [comment]
note: @oliviatessa see #1 for some circumstantial information

Generally: you don't need to import the libraries (`numpy`,`scipy`,`matplotlib`, etc.) every time as long as you run the first block at least once on the opening of the notebook.

* 1a-c. Not really any words here, might want to flesh it out a bit more; same in 2a.

* 2b. Some really large bins in the `df = 1` case which don't really show what the distribution is doing very well. Also, you may want to re-run this section anyways since the plot appears to have randomness consistent with a lower number of samples than when it is rerun with 100k samples.

* 3c. I believe that the probability is well defined in the range `[12.5, inf]`, it's just a clumsy improper integral to calculate.

* 3d. Missing space caused by inadequate padding in line 17

Noncontinuous question:
* 1a. Extra backslash is making your μ not show up properly
* 1b. Can use μ instead on the plot legend of "mu" to be more consistent with other greek letters
At the end, an extra empty block.
* Not exactly sure that the number of rainy days is completely appropriate for a Poission distribution, even though it is an event that happens over a period of time. Since a single day can only rain or not rain (and you can't count two rainy days if it rains twice in a single day) I think a binomial distribution might be more appropriate, since a single day has some probability *p* (152/365) ~= .43, a number of samples (365), and events which can be approximated in general to be independent of others (though in practice this is not entirely true). 
I would presume a question like "how many times does a defined area get hit by a raindrop in an hour" is one more apt for a Poisson, as the probability is infinitesimal while the number of chances is very large.

Other than that, seems like the graphs are all correct and properly ordered, and none of the explainations look wrong after a cursory glance.

File: Lab1.html [hunk 1]
```diff
@@ -0,0 +1,13859 @@
...
+fieldset[disabled] .btn-info.focus {
+  background-color: #5bc0de;
+  border-color: #46b8da;
+}
+.btn-info .badge {
+  color: #5bc0de;
+  background-color: #fff;
+}
+.btn-warning {
+  co
```
Comment (oliviatessa) [inline Lab1.html:13140]: Add more explanation for this first part of the lab. Write as if you are trying to explain these concepts to someone who is unfamiliar with statistical ideas. What is a normal distribution? What is a 'sigma'?

Comment (oliviatessa) [reply Lab1.html:13140]: Remember to interpret your results. How does adjusting sigma affect the distribution?

Comment (oliviatessa) [inline Lab1.html:13536]: Sigma is a quantity related to the standard normal distribution. This is not how you'd find the associated sigma of a chi-squared dist.

**Comment by oliviatessa** [comment]
Overall, your lab needs a lot more explanation (especially the beginning). Think of labs as an essay. You are graded primarily on the quality of your explanations and interpretations of your results. Try to write as if to someone who does not have a background in statistics or know what your assignment is until they read your lab. 

**Also, I think you need to revisit the concept of 'sigma' as a probability.**

**Pull request closed without merging**